app.go

```diff
@@ -369,9 +383,21 @@ func (bn brightcoveNotifier) prettyPrint() string {
 }
 
 func (bc brightcoveConfig) prettyPrint() string {
-	return fmt.Sprintf("\n\t\taddr: [%s]\n\t\toauthAddr: [%s]\n\t\taccountID: [%s]\n\t", bc.addr, bc.oauthAddr, bc.accountID)
```

Comment:
Can you please make sure you don't log sensitive information like tokens and auth details? Otherwise they'll end up in splunk.